### ISSUE TITLE
Support Keyboard Navigation - Modals

### ISSUE THREAD

**Issue by larenelg** [datalust/seq-tickets#969]
Fix keyboard navigation of modals and alerts, currently not accessible with just the keyboard.

**Comment by nblumhardt** [comment]
Done!
